shopelectro/tests/tests_views.py

```diff
@@ -415,9 +415,11 @@ def test_autocomplete_has_no_model_pages(self):
         response = self.client.get(
             f'{reverse("autocomplete")}?term={self.QUOTED_SIGNLE_RESULT_TERM}'
         )
-        result = list(filter(lambda d: d['type'] == 'category', json_to_dict(response)))
-        self.assertTrue(
-            len(result) == 1
+        self.assertFalse(
+            list(filter(
+                lambda r: r['type'] != 'category' and r['type'] != 'see_all',
```

Comment:
`r['type'] not in ['category', 'see_all']`.
It'll be good to compare with type 'page' (or 'page_model') explicitly. It's technically impossible?